### ISSUE TITLE
Support multiline hex literals.

### ISSUE THREAD

**Issue by junderw** [solidity-parser/parser#32]
This was merged in https://github.com/ethereum/solidity/pull/7524 1 year ago.

Given the case:

```solidity
string memory a = 'abc' 'def'
   'ghi'     'jkl';

string memory b = 'abcdefghijkl';

bytes memory c = hex'dead' hex'beef'
     hex'deadbe' hex'ef';

bytes memory d = hex'deadbeefdeadbeef';
```

`a == b` and `c == d` in this case. line breaks and spaces can be in between each section.

It makes code readability much easier.

We use prettier, so doing this breaks formatting for us.

**Comment by junderw** [comment]
It looks like solidity works even with comments in between the sections:

```solidity
string memory a = 'abc' /* test */ 'def'
  // test
   'ghi'     'jkl';

string memory b = 'abcdefghijkl';

bytes memory c = hex'dead' /* test */ hex'beef'
  // test
     hex'deadbe' hex'ef';

bytes memory d = hex'deadbeefdeadbeef';
```
These all compile and work fine.

**Comment by junderw** [comment]
Looks like StringLiterals are supported for multiline/spaces.

If you could add support to HexLiterals in a similar manner I would appreciate it. Thanks.

**Comment by fvictorio** [comment]
Fixed in 0.9.1.
